src/mui/input/RichTextInput.js

```diff
@@ -1,30 +1,39 @@
+import debounce from 'lodash.debounce';
 import React, { Component, PropTypes } from 'react';
 import Quill from 'quill';
 
 require('./RichTextInput.css');
 
 class RichTextInput extends Component {
-    init(container) {
-        if (!container || this.quill) {
-            return;
-        }
-
-        this.quill = new Quill(container, {
-            modules: {
-                toolbar: true,
-            },
+    componentDidMount() {
+        const { toolbar } = this.props;
+        this.quill = new Quill(this.divRef, {
+            modules: { toolbar },
             theme: 'snow',
         });
+
+        const { record, source } = this.props;
+        this.quill.pasteHTML(record[source]);
```

Comment:
No, as it results in a strange behavior: some keys are erased between record update and editor re-render. Probably because of some delay in Quill manipulations.